src/chttpd/src/chttpd_node.erl

```diff
@@ -117,6 +117,12 @@ handle_node_req(#httpd{method='GET', path_parts=[_, Node, <<"_stats">> | Path]}=
     chttpd:send_json(Req, EJSON1);
 handle_node_req(#httpd{path_parts=[_, _Node, <<"_stats">>]}=Req) ->
     send_method_not_allowed(Req, "GET");
+handle_node_req(#httpd{method='GET', path_parts=[_, Node, <<"_prometheus">>]}=Req) ->
+    Metrics = call_node(Node, couch_prometheus_server, scrape, []),
+    Header = [{<<"Content-Type">>, <<"text/plain">>}],
```

Comment:
I think there's supposed to be a version parameter on this?

Reply:
I don't think there is: https://en.wikipedia.org/wiki/List_of_HTTP_header_fields

Reply:
I'm referring to https://prometheus.io/docs/instrumenting/exposition_formats/

`HTTP Content-Type | text/plain; version=0.0.4 (A missing version value will lead to a fall-back to the most recent text format version.)`